datamade/committee-oversight
# Initial Wagtail setup

## Overview

Closes #62 

This PR installs Wagtail and sets up what we'll need for a persistent media directory using lessons learned in [LISC](https://github.com/datamade/lisc-cnda/pull/76) and now documented in [deploy-a-site](https://github.com/datamade/deploy-a-site/blob/master/File-uploads.md). It also creates a first sample data fixture of CMS content.  


## Testing Instructions

* Follow instructions in the [README](https://github.com/datamade/committee-oversight) to get the site running locally
* Run migrations: `python manage.py migrate`
* Create a superuser with a username and password of your choice: `python manage.py createsuperuser`
* Load initial CMS content: `python manage.py load_cms_content`
* Run the app locally: `python manage.py runserver`
* Navigate to the CMS login: http://localhost:8000/cms/
* Make sure you see a Wagtail login screen and that your new user login works!
* Make sure the CMS content loaded property. You should see: 
	- A single page with the title "Committee Oversight Database"
	- This cow under the Images tab on the left sidebar:

![flowercow](https://user-images.githubusercontent.com/1094243/64386351-77333c00-cffe-11e9-8949-5a27c6c4d5d2.jpg)
